src/sections/charts/index.js

```diff
@@ -1,9 +1,201 @@
+                    <div className="react-rainbow-admin-charts_header">
+                        <h2 className="rainbow-p-bottom_medium">Pie Charts</h2>
+                        <ButtonGroup>
+                            <Button variant="neutral" className="react-rainbow-admin-chart_button">
+                                <FontAwesomeIcon icon={faPlus} className="rainbow-m-right_small" />
+                            Color
+                            </Button>
+                            <Button variant="neutral" className="react-rainbow-admin-chart_button">
+                                <FontAwesomeIcon icon={faMinus} className="rainbow-m-right_small" />
+                            Color
+                            </Button>
+                        </ButtonGroup>
+                    </div>
+                    <p className="react-rainbow-admin-charts_chart-title rainbow-align-content_center rainbow-font-size-text_x-small rainbow-color_gray-3">
+                        COLORS USE
+                    </p>
+                    <Chart className="react-rainbow-admin-charts_card rainbow-p-top_x-small" labels={['Yellow', 'Green', 'Red', 'Orange']} type="pie" legendPosition="right">
+                        <Dataset key="Red" title="Red" values={[80, 145, 45, 90]} backgroundColor={['#ffcc00', '#1ad1a3', '#fe4849', '#ff6837']} />
+                    </Chart>
+                </Card>
+                <Card className="react-rainbow-admin-charts_card rainbow-p-vertical_medium rainbow-p-horizontal_small">
+                    <div className="react-rainbow-admin-charts_header">
+                        <h2 className="rainbow-p-bottom_medium">Doughnut Charts</h2>
+                        <ButtonGroup>
+                            <Button variant="neutral" className="react-rainbow-admin-chart_button">
+                                <FontAwesomeIcon icon={faPlus} className="rainbow-m-right_small" />
+                            Color
+                            </Button>
+                            <Button variant="neutral" className="react-rainbow-admin-chart_button">
+                                <FontAwesomeIcon icon={faMinus} className="rainbow-m-right_small" />
+                            Color
+                            </Button>
+                        </ButtonGroup>
+                    </div>
+                    <p className="react-rainbow-admin-charts_chart-title rainbow-align-content_center rainbow-font-size-text_x-small rainbow-color_gray-3">
+                        COLORS USE
+                    </p>
+                    <Chart className="react-rainbow-admin-charts_card rainbow-p-top_x-small" labels={['Yellow', 'Green', 'Red', 'Orange']} type="doughnut" legendPosition="right">
+                        <Dataset key="Red" title="Red" values={[65, 110, 35, 80]} backgroundColor={['#ffcc00', '#1ad1a3', '#fe4849', '#ff6837']} />
+                    </Chart>
+                </Card>
```

Comment:
each card is pretty much the same, so let's create a sub component or two sub components (ChartCard and CircleChartCard) with props like:
- title
- onAddDataSet
- onRemoveDataSet
- etc.
Think about common things and abstract it.
I think would be easier if you pass the Dataset components like children since pass to much props for a component complicates it.